### PR TITLE
 added alert, avatar and badge components

### PR DESCRIPTION
**Pull request by GeetaPogul**

### CODE REVIEW

File: Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+@import url(../CSS/navbar.css);
+
+@import url(../CSS/slidebar.css);
+
+
+.alert-box{
+    width: 90%;
+    height: 50px;
```
Comment (ravikumar1002) [inline Alert/alert.css:8]: one suggestion, try to using rem instead of px

File: Alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    margin:0.2rem;
+    border-radius: 4px;
+
```
Comment (ravikumar1002) [inline Alert/alert.css:12]: use code formatter

File: Alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.warning{
+    background-color: var(--warning-color);
+}
+
+.outline-primary{
+    border: 1px solid var(--solid-primary-color);
+    color: var(--solid-primary-color);
+    background-color: white;
```
Comment (ravikumar1002) [inline Alert/alert.css:35]: use color variable

File: Alert/alert.html [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+        <div class="code-content-div"> 
+       
+            <pre>
+              <code class="code">
+      
+               
+            <-- css -->
+
+
+            .primary{
```
Comment (ravikumar1002) [inline Alert/alert.html:188]: I don't know if it is working fine or not but take a look

File: Avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+    align-items: center;
+    padding: 1.2rem 1.5rem;
+    font-size: 1.2rem;
+    list-style: none;
+   
+    
+}
+.list-links:hover{
+    /* background: #6366F1; */
+    background:#ede9fe;
```
Comment (ravikumar1002) [inline Avatar/avatar.css:39]: use color variable

File: Avatar/avatar.css [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+    margin-bottom: 2rem;
+    font-size: 1.1rem;
+    font-weight: 400;
+   color:var(--content-color);
+}
+
+hr{
+    margin-bottom: 2rem;
+}
+img{
```
Comment (ravikumar1002) [inline Avatar/avatar.css:76]: don't select any html tag directly

**Review by ravikumar1002 — COMMENTED** [review]
some suggestions that you have to apply in your this branch 
- use color variable everywhere
- use code formatter in every file
- if not needed than you can remove comment 
- try to use rem
-  provide some description also

**Review by ravikumar1002 — COMMENTED** [review]
your class name is good

**Review by pratik-kumar-sahu — APPROVED** [review]
- make the required changes as per comments above.
- Looks good to me please go ahead with merging.

**Review by ravikumar1002 — APPROVED** [review]

**Pull request merged**